Corale.Colore/Core/Color.cs

```diff
@@ -365,5 +466,19 @@ public override int GetHashCode()
         {
             return (int)_value;
         }
+
+        /// <summary>
+        /// Clamp a value to 0-255
+        /// </summary>
+        /// <param name="i">The value to be clamped.</param>
+        /// <returns>A value between 0 and 255.</returns>
+        private static int Clamp(int i)
```

Comment:
I think this could be moved to its own static utility class (something like `Corale.Colore.Math`) and made generic (`value, min, max`). It also needs unit tests.